conformance/utils/config/timing.go

```diff
@@ -38,18 +38,25 @@ type InferenceExtensionTimeoutConfig struct {
 	// GatewayObjectPollInterval is the polling interval used when waiting for a Gateway object to appear.
 	GatewayObjectPollInterval time.Duration
 
+	// HTTPRouteConditionTimeout represents the maximum time to wait for an HTTPRoute to have a specific condition.
+	HTTPRouteDeletionReconciliationTimeout time.Duration
+
 	// HTTPRouteDeletionReconciliationTimeout is the time to wait for controllers to reconcile
 	// state after an HTTPRoute is deleted, before checking dependent resources or traffic.
-	HTTPRouteDeletionReconciliationTimeout time.Duration
+	HTTPRouteConditionTimeout time.Duration
 }
 
 // DefaultInferenceExtensionTimeoutConfig returns a new InferenceExtensionTimeoutConfig with default values.
 func DefaultInferenceExtensionTimeoutConfig() InferenceExtensionTimeoutConfig {
+	// Create a local timeout config to modify some values for Inference specific scenarios.
+	modifiedTimeoutConfig := gatewayconfig.DefaultTimeoutConfig()
+	modifiedTimeoutConfig.HTTPRouteMustHaveCondition = 5 * time.Minute // Increased timeout to 5 minutes
```

Comment:
not sure why is this needed? if the defaultTimeoutConfig is not sufficient worth changing it or having a differentTimeout config for inference testing?